addons/website_forum/static/src/js/tours/website_forum.js

```diff
@@ -57,7 +57,7 @@
         trigger: ".modal-header button.btn-close",
         auto: true,
     }, {
-        trigger: ".o_wforum_validate_toggler[data-karma]:first",
-        content: _t("Click here to accept this answer."),
-        position: "right",
+        trigger: ".o_wforum_answer",
+        content: _t("Check the answer"),
+        isCheck: true,
```

Comment:
Don't change the wording of the step, this is visible to users through real tour (not only test).
I am not sure what would be the best solution for those tours requiring a user click as last step.
I guess it should be about adding a next step which would be run only in test mode and not "user mode" with `auto: true`.
In this case, restore the step that you changed, and add a new step after which would check that the "Click here to accept this answer" ran correctly.

```js
{
    trigger: ".something-appearing-after-accepting-the-answer"
    auto: true,
    isCheck: true,
}
```

Reply:
Hi @rdeodoo ,

Actually, Performing the last step: `_t("Click here to accept this answer.")` produces a warning on the UI side (see screenshot). I thought it was actually expected behaviour So I changed the last step instead of adding a check for a warning. But I was wrong there, I just researched more and it's working fine for 16.3 and below. So I guess it's an issue, a warning should not be there.

**Issue**: IMO, Users should able to make their own answers as best on their own Questions (same as 16.3 and below).

Like to have your view on it, is it expected behaviour or not? If not, should we wait for the fix?

![Questions](https://github.com/odoo/odoo/assets/125337508/a0ed05e3-d076-45dd-93c8-4c38fc15b44b)

Reply:
I don't know, it's probably a mistake, but is it relevant to this PR?
At best, you could submit a feedback about it to the related team.

Reply:
Hello @rdeodoo, A [task](https://www.odoo.com/web#id=3481264&cids=2&model=project.task&view_type=form) has been created for the issue. An error message is intended but the message should be something else as mentioned in the task.

> but is it relevant to this PR?

Yes, It's about the last tour step which ignored the action to perform.

Should we wait for the fix or go ahead by just checking the warning( ignoring the error message that appears )?

Reply:
Hello,
The error is not a mistake, we did not want to allow this from 16.4 onward. The tmentionned ask will rephrase the error message.

This step could indeed be updated to simply check that the answer is present, but what would make be even better is checking that the button to edit the answer is present, if we may ask this of you while you are here?

In any case, thank you for the care put into this!

Reply:
![image](https://github.com/odoo/odoo/assets/30048408/cb324e29-cb08-460c-95c9-5bce135d932b)
It's not really what was asked here.. I'll push force the last changes directly so this can be merged